themes/ddbasic/sass/components/node/event.scss

```diff
@@ -257,10 +257,18 @@
 
           &.ticket-available {
             background-color: $green;
+            color: $white;
 
             &:hover {
               background-color: $green-text;
-              color: lighten($black, 15);
+              color: lighten($white, 15);
+            }
+
+            a {
+              &:link, &:visited, &:hover, &:active {
+                color: $white;
+                text-decoration: underline;
```

Comment:
I cannot seem to grok this change. It targets generic links inside `.tickets-available` but that class only seems to be set directly on links on L590. Am I missing something?

Reply:
ticket-available is set on L590 @ modules/ding_place2book/ding_place2book.module  and L309 @ themes/ddbasic/template.node.php, right. What do you think is wrong?

Reply:
@guddo As far as I can tell the change above will result in the following slightly abbreviated selectors:

```css
.button.ticket-available a:link { color: $white; text-decoration: underline; }
``` 

This targets the following HTML element:

```html
<div class="parent">
  <div class="button ticket-available">
    <a href="http://domain.com/">...</a>
  </div>
</div>
```

All the places where the `ticket-available` class is used it seems to be used directly on the link e.g.

```php
$class = 'ticket-available';
$data = l(t('Book a ticket'), $event->links->sales_location, array(
  'attributes' => array(
    'target' => '_blank',
    'class' => array('button', $class),
  ),
));
```

I assume this results in the following markup which is not targeted by the above selectors.

```html
<a class="button ticket-available" href="http://domain.com/">...</a>
```

If that is correct then the change above does not seem to do anything. I am not sure whether it needs to be updated or removed.

Reply:
You are on to something here. It's quite hard for us to test now that we don't have P2b credentials and events with various states anymore. @holt83 do you have a running environment and able to check it out?

Reply:
Yes, I will have a look at it tomorrow.

I was planning of doing a thorough test of all the states again after these changes anyway :)

Reply:
@guddo, @kasperg Kasper is right: the selector does not apply, since the classes is added directly to the anchor tag. 

The text color IS white, because you already set that [further up](https://github.com/ding2/ding2/pull/1405/files#diff-bbdbc25e57d7a11faa06cf1676d62b5bR265), but the text is not underlined.